### DISCUSSION TITLE
How do I enable backspace?

### DISCUSSION THREAD

**Discussion by greasycrow** [blinksh/blink#1986] [Q&A]
I'm using Blink to ssh in to a online community with a command line interface on iPad or iPhone.
If I make a typo and try to backspace I get ^?^?^
I have to hit return to go back to the prompt and try again.
In the settings file, I've set the erase character as:
 stty erase ^h
which I thought would control it, but it doesn't.
Any suggestions? Thanks.

**Comment by carloscabanero** [comment]
Dupe of https://www.reddit.com/r/BlinkShell/comments/1bc6bzd/enable_backspace/ - responding there.
